src/content/learn/updating-arrays-in-state.md

```diff
@@ -782,8 +855,8 @@ Behind the scenes, Immer always constructs the next state from scratch according
 - Instead of mutating an array, create a *new* version of it, and update the state to it.
 - You can use the `[...arr, newItem]` array spread syntax to create arrays with new items.
 - You can use `filter()` and `map()` to create new arrays with filtered or transformed items.
+- Use `toReversed()` or `toSorted()` instead of `reverse()` and `sort()`
```

Comment:
This needs extensions on the prior bullet points too to mention toSpliced() for instance.